README.rst

```diff
@@ -9,16 +9,16 @@ Contributing
 
 This project only requires that:
 
-#. you have `tox`_ installed
-#. you have Python 3 installed locally
+#. you have `sphinx`_ installed (`pip install sphinx`)
 
 To contribute to this repository, fork the project and make your changes.
-Before submitting a pull request, please check that your changes render as 
+Before submitting a pull request, please check that your changes render as
 expected by running::
 
-    $ tox -e docs,serve
+    $ make html
```

Comment:
We can add that they can install tox in a virtualenv or otherwise. It works both ways. The Makefiles apparently (I haven't used one in ages) do not.